lambdas/handlers/document_manifest_by_nhs_number_handler.py

```diff
@@ -33,68 +29,34 @@
 @override_error_check
 def lambda_handler(event, context):
     request_context.app_interaction = LoggingAppInteraction.DOWNLOAD_RECORD.value
-
-    logger.info("Starting document manifest process")
```

Comment:
Our best practices doc asks us to include an entry point log so we can check the handler is reachable. I feel this should go back in, or something a bit more explicit  about what's running if you're feeling spicy

Reply:
It was removed due to duplication on line 37